### DISCUSSION TITLE
New Faction - Zorxian Technocracy

### DISCUSSION THREAD

**Discussion by WolfTooth-2024** [endless-sky/endless-sky#10439] [Ideas]
Looking to develop a new faction and need some feedback regarding what I have so far:

Tier: ???
Former organic race currently in the process of becoming a race of synthetic beings.

Technocracy of Zorx will be the main faction. Has control of the majority of the factions star systems and can be considered the “official” government for the race. They will be the main faction the player wants to deal with if they want the majority of Zorxian technology, ships, equipment, and plot. They are currently in an era of exploration and expansion into a uncharted region of space connected to their home-region by a recently discovered wormhole. They are primarily seeking out new Orvidium and rare metal deposits to expand the production of synthetic frames for the remainder of their population. Not inherently evil, the government is currently strapped for resources and simply can not meet all the demand for new frames and construction demanded on it by this period of great transformation, exploration, and expansion. Desperate times call for desperate measures and the Technocracy was forced to cancel the production of all spare parts for older transfers until the production of new frames is completed and the entire (willing) population is transferred. 

Outmodes will be a sub-faction and represents the various outmodes who upgraded in the past but are now destitute, falling apart, and lacking the spare parts needed to repair themselves combined with a large chunk of the non-transfers – individuals who have not yet acquired their new synthetic body either because they have rejected the chance to upgrade or simply have not had a chance to upgrade due to the lack of synthetic frames to go around. Most Outmodes have lost their livelihoods as the restructuring of the economy has made the demand for certain careers plummet and have turned to lives of crime. Outmode stations are notorious for selling cheap, sub-standard replacement parts for other outmodes, illegal weapon attachments, and hacking and slicing tools in addition to often serving as ports for Smugglers and Pirates. Outmodes are far more de-centralized than the Technocracy resulting in their systems being far more spread apart.

General Starship Design:
•    High hull hit points on all ships.
•    Auto-Repair Systems with significant out of combat repair.
•    Repair Drones with some in-combat repair.
•    Strong anti-small craft and point defense weaponry.

•    Above Average – Decent Speed.

•    Very Poor Shield HP Values and shield regen
•    Below Average-Poor Anti Large Ship Weaponry
•    Usually fairly poor maneuverability.

I plan on posting more in this thread as I get additional ideas/work done in addition to responsding to any feedback you all might have. Thank you in advance.

**Comment by TheGiraffe3** [comment]
Is there a place we can see what you've got so far?
